eric.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Eric's  Page</title>
+</head>
+<body>
+  <img src="http://i.giphy.com/ZL0pekouS3QI0.gif">
+</body>
+<h1>
+  Learn-Git ERIC
+</h1>
+<h2>
+  Slack <name:>scoobaroo</name:>
```

Comment:
Not sure, atom autofilled for me


On Mon, Mar 14, 2016 at 4:30 PM, tgaff <notifications@github.com> wrote:

> In eric.html
> <https://github.com/sf-wdi-27-28/learn-git/pull/27#discussion_r56094218>:
>
> > @@ -0,0 +1,18 @@
> > +<!DOCTYPE html>
> > +<html>
> > +<head>
> > +  <title>Eric's  Page</title>
> > +</head>
> > +<body>
> > +  <img src="http://i.giphy.com/ZL0pekouS3QI0.gif">
> > +</body>
> > +<h1>
> > +  Learn-Git ERIC
> > +</h1>
> > +<h2>
> > +  Slack <name:>scoobaroo</name:>
>
> Is name: a real HTML tag?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sf-wdi-27-28/learn-git/pull/27/files#r56094218>.
>